pkgs/servers/rpiplay/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib, stdenv, pkg-config, fetchFromGitHub, fetchpatch, cmake, wrapGAppsHook, avahi, avahi-compat, openssl, gst_all_1, libplist }:
+
+stdenv.mkDerivation rec {
+  pname = "rpiplay";
+  version = "2021-04-29";
+
+  src = fetchFromGitHub {
+    owner = "FD-";
+    repo = "RPiPlay";
+    rev = "4a65c7b84ff09acd7d6d494bb9037bbf106153a8";
+    sha256 = "06znqlj5b09z3mc965dyvmx2kh64fd7s571q34fh8r9ng98k351s";
+  };
+
+  patches = [
+    # allow rpiplay to be used with firewall enabled.
+    # sets static ports 7000 7100 (tcp) and 6000 6001 7011 (udp)
+    (fetchpatch {
+      name = "use-static-ports.patch";
+      url = "https://patch-diff.githubusercontent.com/raw/FD-/RPiPlay/pull/196.patch";
```

Comment:
Please fetch the commit instead of the PR. The PR can still change which would invalidate the hash.

Reply:
done